apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.Tests/UrlSignerTest.OptionsTest.cs

```diff
@@ -240,6 +241,45 @@ public void WithPort_Null()
                 Assert.NotSame(options, newOptions);
                 Assert.Null(newOptions.Port);
             }
+
+            [Fact]
+            public void WithDefultOverrides_NullOverrideValues()
```

Comment:
Defult => Default here and in the other tests

(And while amending this commit, the message is missing an "r" from "URLSigner".)